plugins/richcombo/plugin.js

```diff
@@ -318,6 +319,14 @@ CKEDITOR.plugins.add( 'richcombo', {
 
 					textElement.setText( typeof text != 'undefined' ? text : value );
 				}
+
+				var valueLabel = typeof text != 'undefined' ? text : value,
+					valueLabelElement = this.document.getById( 'cke_' + this.id + '_value_label' ),
```

Comment:
There are now three places that use `_value_label`. From my point of view, it could be useful to create some helper function which returns a properly formatted id for this element. OTOH, it may end with a big amount of functions like this. However, to change behaviour that relays on template values you don't have to search and alter so many places.

Could we add this kind of 'getter' here?